app/src/main/AndroidManifest.xml

```diff
@@ -20,7 +20,7 @@
     <application
         android:allowBackup="true"
         android:name=".App"
-        android:icon="@mipmap/ic_launcher_48dp"
+        android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
```

Comment:
`android:roundIcon="@mipmap/ic_launcher_round"` is missing here for devices < API 26 and launchers supporting it